packages/eslint-plugin-svelte/src/rules/prefer-const.ts

```diff
@@ -0,0 +1,99 @@
+import { createRule } from '../utils';
+import type { TSESTree } from '@typescript-eslint/types';
+
+import { defineWrapperListener, getCoreRule } from '../utils/eslint-core';
+
+const coreRule = getCoreRule('prefer-const');
+
+type Declaration = TSESTree.Expression;
+
+/**
+ * Finds and returns the callee of a declaration node within variable declarations or object patterns.
+ */
+function findDeclarationCallee(node: Declaration) {
+	const { parent } = node;
+	if (parent.type === 'VariableDeclarator' && parent.init?.type === 'CallExpression') {
+		return parent.init.callee;
+	}
+
+	if (
+		parent.type === 'Property' &&
+		parent.parent.type === 'ObjectPattern' &&
+		parent.parent.parent.type === 'VariableDeclarator' &&
+		parent.parent.parent.init?.type === 'CallExpression'
+	) {
+		return parent.parent.parent.init.callee;
+	}
+
+	return null;
+}
+
+/**
+ * Determines if a declaration should be skipped in the const preference analysis.
+ * Specifically checks for Svelte's state management utilities ($state, $props, $derived).
+ */
+function shouldSkipDeclaration(declaration: Declaration | null) {
+	if (!declaration || !declaration.parent) {
+		return false;
+	}
+
+	const callee = findDeclarationCallee(declaration);
+	if (!callee) {
+		return false;
+	}
+
+	if (callee.type === 'Identifier' && ['$state', '$props', '$derived'].includes(callee.name)) {
+		return true;
+	}
+
+	if (callee.type !== 'MemberExpression' || callee.object.type !== 'Identifier') {
+		return false;
+	}
+
+	if (
+		callee.object.name === '$derived' &&
+		callee.property.type === 'Identifier' &&
+		callee.property.name === 'by'
+	) {
+		return true;
+	}
+	if (
+		callee.object.name === '$state' &&
+		callee.property.type === 'Identifier' &&
+		callee.property.name === 'frozen'
```

Comment:
I think it's `$state.raw`.

```suggestion
		callee.property.name === 'raw'
```